java/jni/helper.cxx

```diff
@@ -117,9 +117,63 @@ bool from_jboolean_to_bool(jboolean val)
     return result;
 }
 
+std::string from_jstring_to_string(JNIEnv *env, jstring str)
+{
+    jboolean is_copy = JNI_TRUE;
+    const char *str_chars = (char *)env->GetStringUTFChars(str, &is_copy);
+    if (!str_chars)
+    {
+        throw std::runtime_error("GetStringUTFChars returned NULL\n");
+    }
+    const std::string string_to_write = std::string(str_chars);
+
+    env->ReleaseStringUTFChars(str, str_chars);
+
+    return string_to_write;
+}
+
+tinyb::BluetoothType from_int_to_btype(int type)
+{
+    tinyb::BluetoothType result = tinyb::BluetoothType::NONE;
+
+    switch (type)
+    {
+        case 0:
+            result = tinyb::BluetoothType::NONE;
```

Comment:
Just `return tinyb::BluetoothType::x` for each case would've been enough, or summarize the whole function to a `return static_cast<BluetoothType>(type);`